### DISCUSSION TITLE
Terminal emulator, change font size?

### DISCUSSION THREAD

**Discussion by jayjf** [koreader/koreader#9761] [Q&A] [answered]
Is there any way to change the font size in terminal emulator?

**Comment by Frenzie** [comment, marked as answer]
Isn't that right there in the menu somewhere under "open terminal session"? :-)

**Reply by jayjf** [reply]
(A tad embarrassed) Indeed, it is.  More tools, Terminal emulator, Font size

**Reply by jayjf** [reply]
There's also an option to choose which shell. What's available on the kobo?

**Reply by NiLuJe** [reply]
sh, or zsh if you have my bundle of tools installed (spoiler: I haven't tested ZSH, and the fancy prompt might be slow/ugly there).

**Reply by Frenzie** [reply]
By default only busybox sh but I assume most others should work just fine. Dash and Bash do.
